Makefile

```diff
@@ -1,6 +1,11 @@
-# minimal Makefile to run Python unit tests
-# for flux-accounting
+FLUX_INSTALL_PREFIX ?= /usr
+
+dependencies:
+        ${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install -r requirements.txt --user
+
+install: dependencies
+        ${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install --install-option="--prefix=${FLUX_INSTALL_PREFIX}" \
```

Comment:
I ran into this error on pip v20, and this change seemed to fix it:
```
Installing collected packages: flux-accounting
    Running setup.py install for flux-accounting ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-z41xl4kb/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-z41xl4kb/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'
"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-79duizhp/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /home/sherbein/.local/include/python3.8/flux-accountin
g --prefix=/home/sherbein/opt/flux/accounting-pr-19 --install-scripts=/home/sherbein/opt/flux/accounting-pr-19/libexec/flux/cmd/
         cwd: /tmp/pip-req-build-z41xl4kb/
    Complete output (2 lines):
    running install
    error: can't combine user with prefix, exec_prefix/home, or install_(plat)base
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-z41xl4kb/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-z41xl4kb/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__)
;code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-79duizhp/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /home/sherbein/.local
/include/python3.8/flux-accounting --prefix=/home/sherbein/opt/flux/accounting-pr-19 --install-scripts=/home/sherbein/opt/flux/accounting-pr-19/libexec/flux/cmd/ Check the logs for full command output.
make: *** [Makefile:7: install] Error 1
```

```suggestion
        ${FLUX_INSTALL_PREFIX}/bin/flux python -m pip install --prefix=${FLUX_INSTALL_PREFIX} \
```

Also worth noting the deprecation warning which means that `--install-option="--install-scripts` might not work in the near future:
```
DEPRECATION: Location-changing options found in --install-option: ['--prefix', '--install-scripts'] from command line. This configuration may cause unexpected behavior and is unsupported. pip 20.2 will remove support for this functionality. A possible replacement is using
pip-level options like --user, --prefix, --root, and --target. You can find discussion regarding this at https://github.com/pypa/pip/issues/7309.
```